PROTOCOL.md

```diff
@@ -424,6 +429,44 @@ enough to get you started. You can watch the data sent and received from
 the server on a regular connection, or look at the client source code
 for a full list of message types.
 
+**Action requests**
+
+These are how the client sends the player's decisions to the server. Most
+requests are sent with `REQUESTID`. `REQUESTID` is used to identify which
+action the request was intended for to protect from cases of trying to send
+multiple requests for the same action, which would otherwise effect future
+decisions. The first decision of a match has a `REQUESTID` of 1, and each
+successful decision increments the counter by 1.
```

Comment:
It should be noted that `REQUESTID` is specifically a guard against race conditions involving the Cancel button.

It's also optional: You can leave off `REQUESTID` if you haven't clicked the Cancel button recently, which helps if you want to type your choices instead of clicking the button.